repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -877,7 +877,7 @@ public void createAndQueueTask(String termName, String termQName, String termGui
         String taskType = isHardDelete ? UPDATE_ENTITY_MEANINGS_ON_TERM_HARD_DELETE : UPDATE_ENTITY_MEANINGS_ON_TERM_SOFT_DELETE;
         String currentUser = RequestContext.getCurrentUser();
         Map<String, Object> taskParams = MeaningsTask.toParameters(termName, termQName, termGuid);
-        AtlasTask task = taskManagement.createTask(taskType, currentUser, taskParams);
+        AtlasTask task = taskManagement.createTask(taskType, currentUser, taskParams, termQName, termGuid);
```

Comment:
discuss this

Reply:
Removed unique and reference id , introduced classification based params